packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/job-logs.controller.js

```diff
@@ -37,21 +39,19 @@ export default class {
   }
 
   downloadLogs() {
-    const re = /https:\/\/storage\.[a-z0-9]+\.cloud.ovh.net\/v1\/AUTH_[a-z0-9]+\/(.*)\/(.*)/;
-    const logsUrl = this.logger.logs.logsAddress;
-    if (logsUrl) {
-      const matches = logsUrl.match(re);
-      this.logger.downloadObject(
-        this.projectId,
-        this.job.region,
-        matches[1],
-        matches[2],
-      );
-      this.isDownloadButtonDisabled = true;
```

Comment:
It looks like `isDownloadButtonDisabled` is no more used meaning that you can clean it in the actual controller but also from the template part.

see: https://github.com/ovh/manager/blob/develop/packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/job-logs.html#L28

Reply:
Thanks for the feedback. It is done :)